jobs/focus_event_longitudinal_view.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
Is this a linter thing? I'm wary of having unrelated changes in this PR (same with the perms changes below and the weird double shebang.) Could you submit these in a separate PR?